Changelog.md

```diff
@@ -62,6 +63,7 @@
   and a document encoding is not an UTF-8
 - [#434]: Fixed incorrect error generated in some cases by serde deserializer
 - [#445]: Use local name without namespace prefix when selecting enum variants based on element names
+  in a serde deserializer
```

Comment:
This should be part of #445, not this PR.

Reply:
Yes, I think I'll split up this to a dedicated PR with other small stuff